src/target/llvm/codegen_nvptx.cc

```diff
@@ -254,14 +254,35 @@ inline int DetectCUDAComputeVersion() {
   }
 }
 
-runtime::Module BuildNVPTX(IRModule mod, std::string target) {
+static void UpdateTargetConfig(const String& key, const String& value,
```

Comment:
If this function can be used, we should move it to be a target utility function.

Reply:
Good idea. I moved it to `build_common.h`